Project-Aurora/Project-Aurora/Devices/OpenRGB/OpenRGBDevice.cs

```diff
@@ -130,14 +130,14 @@ private void UpdateDeviceList()
                     helper.ProcessMappings(fallbackKey);
                     _devices.Add(helper);
                 }
-                Thread.Sleep(500);
-            }
+            Thread.Sleep(500);
+            _updateLock.Release();
```

Comment:
This lock is now includes 500ms sleep. Device update change will be affected by this.

Reply:
I only changed from simple `lock` to `SemaphoreSlim` because `lock` statements can't be used in async methods. The logic is the same, it's just that there's no code block `{ }`. 
It used to be: 
```
lock {
  foreach { }
  Sleep
endlock }
```
Now it's
```
Semaphore.Wait()
  foreach { }
  Sleep
Semaphore.Release()
```